src/mkindex_misc.hpp

```diff
@@ -87,7 +85,7 @@ _readMappingFileUniProt(TInputView                                            &
             taxIdIsPresent[idNum] = true;
         }
 
-        seqan3::detail::consume(fiv | seqan3::detail::take_line);
+        seqan3::detail::consume(fiv | std::views::take_while([](auto && chr){ return chr != '\n'; }));
```

Comment:
This still needs to check for `== '\r'`.

You can also add this somewhere globally:
```cpp
inline constexpr auto not_eol = !(seqan3::is_char<'\r'> || seqan3::is_char<'\n'>);
```
And then just use `std::views::take_while(not_eol)` in all these places

Reply:
Yes, sorry I missed that ..